src/test/mockData/userMock.js

```diff
@@ -2,8 +2,24 @@ export default {
   validUser: {
     firstName: 'Annie',
     lastName: 'Skywalker',
-    email: 'lordvader@order66.com',
+    email: 'lordvader@gmail.com',
```

Comment:
Does the validation fail if you use order66.com?

Reply:
yes it does